draft-irtf-cfrg-frost.md

```diff
@@ -620,26 +602,26 @@ The Coordinator MUST verify the set of signature shares using the following proc
   - index, Index `i` of the signer. Note index will never equal `0`.
   - PK, the public key for the group
   - PK_i, the public key for the ith signer, where PK_i = ScalarBaseMult(s[i])
-  - z_i, the signature share for the ith signer, computed from the signer
-  - R_i, the commitment for the ith signer, computed from the signer
+  - sig_share, the signature share for the ith signer, computed from the signer
+  - comm_share, the commitment for the ith signer, computed from the signer
   - R, the group commitment
   - msg, the message to be signed
   - participant_list, a set containing identifiers for each signer, similarly of length
     NUM_SIGNERS (sent by the Coordinator).
 
-  Outputs: 1 if the signature share is valid, and 0 otherwise
+  Outputs: 1 if the signature share is valid, and 0 otherwise.
 
-  def verify_signature_share(index, PK, PK_i, z_i, R_i, R, msg, participant_list):
+  def verify_signature_share(index, PK, PK_i, sig_share, comm_share, R, msg, participant_list):
     msg_hash = H3(msg)
     group_comm_enc = G.SerializeElement(R)
     group_public_key_enc = G.SerializeElement(group_public_key)
     challenge_input = group_comm_enc || group_public_key_enc || msg_hash
-    c = H2(challenge_input)
+    c = H(challenge_input)
```

Comment:
Needs to be `H1/H2/H3`

Reply:
Thanks, yeah, I forgot to undo this change!